kong/plugins/httplog/log.lua

```diff
@@ -0,0 +1,19 @@
+local cjson = require "cjson"
+local http_client = require "kong.tools.http_client"
+
+local APPLICATION_JSON_TYPE = "application/json"
+
+local _M = {}
+
+local function log(premature, conf, message)
+  local response, status, headers = http_client.post(conf.logging_url, cjson.encode(message), { ["content-type"] = APPLICATION_JSON_TYPE })
```

Comment:
You shouldn't restrict it either, it should probably be a default but you never know. Maybe @ahmadnassri agrees with me on that.